test/learn.jl

```diff
@@ -303,11 +326,9 @@ end
 
     # Test NaN spearman due to unranked input
     votes = [1; 2; 2]
-    predicted_soft = [
```

Comment:
...juliaformatter got hold of these matrices, and I didn't have the heart to undo the updates.